### PR TITLE
Update Glut window to use glutDestroyWindow

### PR DESCRIPTION
**Pull request by silktown-software**
Section 3.8 Escape key handling was incorrectly using `sys.exit()` to close the window and will error when attempted to close the process. `glutDestroyWindow` needs to be called instead. Updated example code accordingly.

https://github.com/rougier/python-opengl/issues/64

### CODE REVIEW

**Comment by rougier** [comment]
There are a lot of changes in your PR aprat from the `glutDestroyWindow`, any reason?

File: 03-quickstart.rst [hunk 1]
```diff
@@ -200,29 +200,34 @@ package. Here is a minimal setup that should open a window with a black backgrou
...
 
-   import sys
    import OpenGL.GL as gl
    import OpenGL.GLUT as glut
 
+   window = None
+
+
    def display():
-       gl.glClear(gl.GL_COLOR_BUFFER_BIT)
```
Comment (rougier) [inline 03-quickstart.rst:208]: Why did you remove the clear?

Comment (silktown-software) [reply 03-quickstart.rst:208]: I am not sure now. I will add it back in an update the PR.

**Comment by silktown-software** [comment]
I've added that glClear back in. I can't remember why I removed it, so in all likely hood it was done in error.
